tests/test_zeros_synth.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import torch
+import os
+
+import pytest
+from scvi.models import VAE
+from scvi.inference import UnsupervisedTrainer
+from scvi.dataset.synthetic import ZISyntheticDatasetCorr
+from sklearn.manifold import TSNE
+import matplotlib.pyplot as plt
+import seaborn as sns
+sns.set()
+
+
+def test_enough_zeros():
+    """
+    Can be seen as a 'pre' test for test_zeros_classif
+    In test_zeros_classif, we classify the zeros obtained by scVI
+    Hence, the objective of test_enough_zeros is to check that the synthetic
+    dataset has enough zeros and that the proportion of technical zeros over biological zeros
+    is somehow balanced
+    :return:
+    """
+    nb_data = ZISyntheticDatasetCorr(n_clusters=8, n_genes_high=15, n_overlap=3,
+                                     lam_0=320, n_cells_cluster=100,
+                                     weight_high=1.714286, weight_low=1,
+                                     dropout_coef_high=0.08, dropout_coef_low=0.05)
+
+    print(nb_data.X.shape)
+    print(nb_data.exprs_param.min(), nb_data.exprs_param.max())
+    is_technical_mask = nb_data.is_technical.squeeze()
+    nb_data_zeros = nb_data.X == 0
+    tech_zeros = nb_data_zeros[is_technical_mask].sum()
+    bio_zeros = nb_data_zeros[~is_technical_mask].sum()
+    print("Prop of technical zeros :", tech_zeros)
+    print("Prop of biological zeros :", bio_zeros)
+    assert .2 <= tech_zeros / float(bio_zeros) <= 5.
+    assert tech_zeros >= 1000
+
+
+@pytest.mark.model_fit
```

Comment:
Please remove the `@pytest.mark.model_fit` decorator too. Instead, set `n_epoch = 1` if model fit wasn't specified (and `n_epoch = 150` otherwise.

Reply:
Normally fixed by 76fc30c